src/sheldon.erl

```diff
@@ -59,17 +59,24 @@ check_words(Words, Config) -> check_words(Words, [], Config).
                  , sheldon_config:config()
                  ) -> [string()].
 check_words([], Acc, _Config) -> Acc;
-check_words([Word | Rest], Acc, Config = #{lang := Lang}) ->
+check_words([Word | Rest], Acc, Config) ->
   Normalized = sheldon_utils:normalize(Word),
-  Acc1 = case is_correct(Lang, Normalized) of
+  Acc1 = case correctly_spelled(Normalized, Config) of
            true  -> Acc;
            false -> [Normalized | Acc]
          end,
   check_words(Rest, Acc1, Config).
 
--spec is_correct(sheldon_dictionary:language(), string()) -> boolean().
-is_correct(Language, Word) ->
-  case sheldon_utils:is_number(Word) of
-    false -> sheldon_dictionary:exist_in_dictionary(Language, Word);
+-spec correctly_spelled(string(), sheldon_config:config()) -> boolean().
+correctly_spelled(Word, Config = #{lang := Lang}) ->
+  case sheldon_utils:is_number(Word) or ignore(Word, Config) of
```

Comment:
I think you should use `orelse` instead of just `or`, that way the code avoid to call `ignore/2` for a number.